tools/chef/cookbooks/px_dev/recipes/golang.rb

```diff
@@ -42,7 +42,6 @@
 execute 'install go binaries' do
   ENV['GOPATH'] = "/opt/px_dev/gopath"
   command %(go install github.com/golang/mock/mockgen@v1.5.0 && \
-            go install github.com/cheekybits/genny@v1.0.0 && \
```

Comment:
Trying to install genny results in the following error:

```
$ go install github.com/cheekybits/genny@v1.0.0
go: finding module for package golang.org/x/tools/imports
go: toolchain upgrade needed to resolve golang.org/x/tools/imports
go: golang.org/x/tools@v0.25.0 requires go >= 1.22.0 (running go 1.21.0)
```

I wasn't able to get to the bottom of how this changed between the beginning on my PR and now or what dependency is causing this toolchain requirement.

Anyway, it appears that the last reference to genny was removed in 806ad3f67623c0f92c63b663ae2978658813e116. This opts to remove it instead of trying to work around this issue.

```
# Verify that genny is no longer used on that commit

ddelnano@dev-vm:~/code/pixie-worktree ((806ad3f67...)) $ git grep -i genny
go.sum:github.com/gobuffalo/genny v0.0.0-20190329151137-27723ad26ef9/go.mod h1:rWs4Z12d1Zbf19rlsn0nurr75KqhYp52EAGGxTbBhNk=
go.sum:github.com/gobuffalo/genny v0.0.0-20190403191548-3ca520ef0d9e/go.mod h1:80lIj3kVJWwOrXWWMRzzdhW3DsrdjILVil/SFKBzF28=
go.sum:github.com/gobuffalo/genny v0.1.0/go.mod h1:XidbUqzak3lHdS//TPu2OgiFB+51Ur5f7CSnXZ/JDvo=
go.sum:github.com/gobuffalo/genny v0.1.1 h1:iQ0D6SpNXIxu52WESsD+KoQ7af2e3nCfnSBoSF/hKe0=
go.sum:github.com/gobuffalo/genny v0.1.1/go.mod h1:5TExbEyY48pfunL4QSXxlDOmdsD44RRq4mVZ0Ex28Xk=
Binary file go_deps.bzl matches
tools/arc_addons/pixielabs/unit/ArcanistGoGenCheckerTestEngine.php:    'genny' => '/(?<=-out )(.*)(?=\.gen\.go)/',
tools/chef/cookbooks/pixielabs/recipes/base.rb:            github.com/cheekybits/genny@v1.0.0 \

# Go back a commit and see that XXX removed the final reference

ddelnano@dev-vm:~/code/pixie-worktree ((806ad3f67...)) $ git checkout HEAD~1
Previous HEAD position was 806ad3f67 Use informers for k8s metadata watcher
HEAD is now at b0d2c6c56 Add dataAccess flag to CLI

ddelnano@dev-vm:~/code/pixie-worktree ((b0d2c6c56...)) $ git grep -i genny
go.sum:github.com/gobuffalo/genny v0.0.0-20190329151137-27723ad26ef9/go.mod h1:rWs4Z12d1Zbf19rlsn0nurr75KqhYp52EAGGxTbBhNk=
go.sum:github.com/gobuffalo/genny v0.0.0-20190403191548-3ca520ef0d9e/go.mod h1:80lIj3kVJWwOrXWWMRzzdhW3DsrdjILVil/SFKBzF28=
go.sum:github.com/gobuffalo/genny v0.1.0/go.mod h1:XidbUqzak3lHdS//TPu2OgiFB+51Ur5f7CSnXZ/JDvo=
go.sum:github.com/gobuffalo/genny v0.1.1 h1:iQ0D6SpNXIxu52WESsD+KoQ7af2e3nCfnSBoSF/hKe0=
go.sum:github.com/gobuffalo/genny v0.1.1/go.mod h1:5TExbEyY48pfunL4QSXxlDOmdsD44RRq4mVZ0Ex28Xk=
Binary file go_deps.bzl matches
Binary file src/vizier/services/metadata/controllers/k8smeta/k8s_metadata_utils.gen.go matches
src/vizier/services/metadata/controllers/k8smeta/k8s_metadata_utils.go://go:generate genny -in=k8s_metadata_utils.tmpl -out k8s_metadata_utils.gen.go gen "ReplacedResource=Pod,Service,Namespace,Endpoints,Node"
src/vizier/services/metadata/controllers/k8smeta/k8s_metadata_utils.tmpl:       "github.com/cheekybits/genny/generic"
tools/arc_addons/pixielabs/unit/ArcanistGoGenCheckerTestEngine.php:    'genny' => '/(?<=-out )(.*)(?=\.gen\.go)/',
tools/chef/cookbooks/pixielabs/recipes/base.rb:            github.com/cheekybits/genny@v1.0.0 \
```